experimental/packages/opentelemetry-instrumentation-http/src/http.ts

```diff
@@ -847,14 +838,30 @@ export class HttpInstrumentation extends InstrumentationBase<HttpInstrumentation
     );
   }
 
+  private _onIncomingResponseError(
```

Comment:
Looks like you are calling this for outgoing requests, can you rename this method?

Reply:
Fixed !